doc/user/sql/create-source/avro-source.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "CREATE SOURCE: Avro over Kafka"
+description: "Learn how to connect Materialize to an Avro-formatted Kafka topic"
+menu:
+  main:
+    parent: 'create-source'
+aliases:
+    - /docs/sql/create-source/kafka
+    - /docs/sql/create-source/avro
+---
+
+`CREATE SOURCE` connects Materialize to some data source, and lets you interact
+with its data as if it were in a SQL table.
+
+This document details how to connect Materialize to an Avro-formatted Kafka
+topic. For other options, view [`CREATE  SOURCE`](../).
+
+## Conceptual framework
+
+Sources represent connections to resources outside Materialize that it can read
+data from. For more information, see [API Components:
+Sources](../../../overview/api-components#sources).
+
+## Syntax
+
+{{< diagram "create-source-avro.html" >}}
+
+Field | Use
+------|-----
+_src&lowbar;name_ | The name for the source, which is used as its table name within SQL.
+**KAFKA BROKER** _host_ | The Kafka broker's host name.
+**FORMAT ...** _url_ | The URL of the Confluent schema registry to get schema information from.
+**FORMAT ...** _schema&lowbar;file&lowbar;path_ | The absolute path to a file containing the schema.
+**FORMAT ...** _inline&lowbar;schema_ | A string representing the schema.
+**ENVELOPE** _envelope_ | The envelope type.<br/><br/> &#8226; **NONE** creates an append-only source. This means that records will only be appended and cannot be updated or deleted. <br/><br/>&#8226; **DEBEZIUM** creates a source that supports all CRUD-like operations. This option requires records have the [appropriate fields](#format-implications), and is generally only supported by sources published to Kafka by [Debezium].<br/><br/>For more information, see [Debezium envelope details](#debezium-envelope-details).
```

Comment:
I think this description is a bit confusing -- when you say "supports all CRUD-like operations" it sounds as though *materialize itself* supports those operations; i.e., someone can update the source by running `UPDATE` and so on at the Materialize console.